src/js/app.js

```diff
@@ -2,6 +2,8 @@
 import 'intersection-observer';
 import Lazzzy from './utils/lazzzy';
 
+var a = 'a';
```

Comment:
__[ESLint]__

Error - 'a' is assigned a value but never used. (no-unused-vars)


<!-- Comment by [SideCI](https://sideci.com) -->